core/domain/exp_services.py

```diff
@@ -1175,9 +1520,14 @@ def get_average_rating(ratings):
 
 
 def get_scaled_average_rating(ratings):
-    """Returns the lower bound wilson score of the ratings as a float. If
-    there are no ratings, it will return 0. The confidence of this result is
-    95%.
+    """Returns the lower bound wilson score of the ratings. If there are
+    no ratings, it will return 0. The confidence of this result is 95%.
+
+    Args:
+        ratings: list(int). List of value of rating.
```

Comment:
Ditto: please check. Look at places that call this function; introspect the value of this arg using print or logging.info or raise Exception(...).